src/mono/mono/mini/interp/transform.c

```diff
@@ -7841,6 +7882,7 @@ generate_code (TransformData *td, MonoMethod *method, MonoMethodHeader *header,
 	}
 
 	g_assert (td->ip == end);
+	mono_bitset_free(il_targets);
```

Comment:
The code here is not guaranteed to run, it should reside at the exit lable.

Reply:
Moved to exit label.